test/end-to-end/helpers/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const Chance = require('chance');
+
+const chance = new Chance();
+
+const helpers = {};
+
+helpers.getAuthClientParams = function() {
```

Comment:
Any reason this is a function instead of just an object literal?

Reply:
No reason, my logic was if we want to change the params based on what test is calling it, it gives us that extensibility. Thoughts?